### ISSUE TITLE
Try it out - Opt-out from auto-filling input fields with examples

### ISSUE THREAD

**Issue by adamgronberg** [swagger-api/swagger-ui#8580]
### Q&A
 - OS: any
 - Browser: any
 - Version: any
 - Method of installation: npm
 - Swagger-UI version: 4.18.2
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration

Swagger/OpenAPI definition:
```yaml
openapi: 3.0.0
info:
  title: Sample API
  version: 1.0.0
servers:
  - url: http://api.example.com/v1
paths:
  /users/{user-id}:
    get:
      summary: Returns a user.
      parameters:
      - name: user-id
        in: path
        description: User id
        required: true
        schema:
          type: string
        example: USER-09324234EXAMPLE
      responses:
        '200':    # status code
          description: A user object
```

### Screenshots

Our user id starts with "USER-XXXXX". This shows the structure of a user-id and is therefore helpful as an example. 
However, this will always fail when using "Try it out". There is, and will never be, a user with id "USER-09324234EXAMPLE"

![Screenshot from 2023-04-20 13-36-22](https://user-images.githubusercontent.com/5450456/233358021-ee3449aa-6bd2-407d-98f5-49fa58b49990.png)

### How can we help?

> tldr; Is there a way to not auto-fil the input fields with example data when the user clicks on “Try it out”

In our OpenAPI specification, we use example values to show how an API should be used and how the input should be formatted. These examples are not valid values, but rather a representation of how the data should look.

However, we have observed that users often send in these example values when using "Try it out" in the Swagger UI. As a result, they get confused when the API responds with an error and then sends a bug report to us.

To prevent this, we would like to disable the auto-filling of parameter/body/query fields with example data, but still show examples (When not using "Try it out"). It is important to ensure that users understand that the examples are not valid values and should not be used in actual requests.

Would a custom plugin solve this for us? Any ideas on how to best implement it? Are others seeing the same issues with users?

---

Note: The exact examples shown here are not our actual use case, but a simplification describing the core issue.

**Comment by sunny34** [comment]
Hi Adam, would you be able to share how did you integrate the above plugin with swagger? I have similar requirement however i am using drf-spectacular to generate open api specification

**Comment by Drakula2k** [comment]
I had to update this plugin code to make it work with the latest version (5.11.3):

```js
  const DontSetParametersDefaultsPlugin = () => ({
    wrapComponents: {
      JsonSchemaForm: (Original, { React }) => ({ value, description, ...props }) => {
        return React.createElement(Original, { ...props, description: value, dispatchInitialValue: true })
      }
    },
  });
```
